core/test/scripts/run-mocha-tests.js

```diff
@@ -277,33 +272,66 @@ function exit({numberFailures, numberMochaInvocations}) {
  * @param {number} invocationNumber
  */
 async function runMocha(tests, mochaArgs, invocationNumber) {
-  process.env.LH_FAILED_TESTS_FILE = `${failedTestsDir}/output-${invocationNumber}.json`;
+  const failedTestsFile = `${failedTestsDir}/output-${invocationNumber}.json`;
+  const notRunnableTestsFile = `${failedTestsDir}/output-${invocationNumber}-nr.json`;
```

Comment:
nit: the `nr` suffix could be confusing and you would need to look into this file to understand what it means. Could we expand to `not-runnable` or perhaps some other suffix?